providers/edge3/src/airflow/providers/edge3/models/edge_worker.py

```diff
@@ -27,13 +27,18 @@
 
 from airflow.exceptions import AirflowException
 from airflow.models.base import Base
+from airflow.providers.edge3.version_compat import AIRFLOW_V_3_1_PLUS
```

Comment:
```suggestion
```

Reply:
This is still needed because it's used later on 

https://github.com/apache/airflow/blob/2f737d2e252e8651f877ee2ab809475db9d8f62e/providers/edge3/src/airflow/providers/edge3/models/edge_worker.py#L173-L225

Reply:
I replaced it with `try-except`.